New Proposed Pull Request Template (#66)
microsoft/code-with-engineering-playbook: Engineering/Templates/PullRequestTemplate.md

Thank you for contributing.
Before submitting this PR, please make sure:

-------------------------------------

> * [ ] Have I followed the guidelines in Contributing as Developers?

Reviewer: ```suggestion
* [ ] I followed the guidelines in [Contributing as Developers](https://github.com/microsoft/code-with-engineering-playbook/blob/master/CONTRIBUTING.md)
```

> * [ ] Have I added an explanation of what my changes do and why I would like you to include them?

Reviewer: ```suggestion
* [ ] I added an explanation of what my changes do and why I would like you to include them
```

> * [ ] I have updated the documentation accordingly.

Reviewer 2: Lines 7-11. Inconsistent line endings (periods). I'd recommend including periods on all items in the checklist.

If for some reason we decide no periods on checklist items, we should remove for consistency.

* [ ] I have added tests to cover my changes.
* [ ] All new and existing tests passed.
* [ ] My code follows the code style of this project.
* [ ] I ran lint checks locally prior to submission.

> * [ ] Have I checked to ensure there aren't other open Pull Requests for the same update/change?

Reviewer: ```suggestion
* [ ] I have checked to ensure there aren't other open Pull Requests for the same update/change
```

> * [ ] Does this introduce a breaking change?

Reviewer: ```suggestion
* [ ] This does not introduce a breaking change
```

## Link to the design document that prompted this PR?
-------------------------------------

> <!-- Please provide a ink to the design document relevant to this area -->

Reviewer: ```suggestion
<!-- Please provide a link to the design document relevant to this area -->
```

> <!-- Please provide a link to the design document relevant to this area -->

Reviewer 2: Missing period to be consistent with other comment statements.

Link to Design Document: 

## What is the current behavior?
-------------------------------------
<!-- Please describe the current behavior that you are modifying, or link to a relevant issue. -->

Issue Number: N/A

## What is the new behavior?
-------------------------------------
<!-- Please describe the behavior or changes that are being added by this PR. -->

-

<!-- If this introduces a breaking change, please describe the impact and migration path for existing applications below. -->

## Any relevant logs, error output, etc?
-------------------------------------

> If it’s long, please upload to (StorageAccount)/pr-support/attachments/(PR Number)/(yourFiles) using (Azure Storage Explorer or portal.azure.com) and insert the link here.

Reviewer: ```suggestion
If it’s long, please upload to (StorageAccount)/pr-support/attachments/(PR Number)/(yourFiles) using [Azure Storage Explorer](https://azure.microsoft.com/en-us/features/storage-explorer/) or [portal.azure.com](portal.azure.com) and insert the link here.
```
